.github/CODEOWNERS

```diff
@@ -669,6 +669,8 @@ peps/pep-0787.rst  @ncoghlan
 # ...
 peps/pep-0789.rst  @njsmith
 # ...
+peps/pep-0791.rst  @mardiros
```

Comment:
Only core developers and triagers can be listed here. Your sponsor (once you have one) should be listed.